docs/HACKING.md

```diff
@@ -1,158 +1,154 @@
-the [Build](#build) step is not required.
 
-In order to build the operator ignoring the existing build image cache, run the above command with the `-i` option.
+#### Set CI to false
+
+This is to tell the tests that this test isn't running in CI.
 
-To clean-up the installation, use:
 ```shell script
-hack/olm.sh cleanup
+export OPENSHIFT_CI=false
 ```
 
-### Running e2e tests on a cluster
-The following environment variables must be set:
+#### Set shared credentials file
+
+If you are running on AWS or GCP, set your shared credentials file.
+
 ```shell script
-# SSH key to be used for creation of cloud-private-key secret
-export KUBE_SSH_KEY_PATH=<path to RSA type ssh key>
-export OPERATOR_IMAGE=quay.io/<insert username>/wmco:$VERSION_TAG
-export OPENSHIFT_CI=false
-# on AWS only:
 export AWS_SHARED_CREDENTIALS_FILE=<path to aws credentials file>
 ```
-Once the above variables are set, run the following script:
-```shell script
-hack/run-ci-e2e-test.sh
+
+```shell script 
+export GOOGLE_CREDENTIALS=<path to google credentials file>
 ```
 
-Additional flags that can be passed to `hack/run-ci-e2e-test.sh` are
-- `-s` to skip the deletion of Windows nodes that are created as part of test suite run
-- `-m` to represent the number of Windows instances to be configured by the Windows Machine controller
-- `-c` to represent the number of Windows instances to be configured by the ConfigMap controller
-- `-b` gives an alternative path to the WMCO binary. This option overridden in OpenShift CI.
-       When building the operator locally, the WMCO binary is created as part of the operator image build process and
-       can be found at `build/_output/bin/windows-machine-config-operator`, this is the default value of this option.
-- `-t` specify the test to run. All tests are run if the option is not used. The allowed options are:
-  - `all` all the tests are run
-  - `basic` creation, network and deletion tests are run
-  - `upgrade` creation, upgrade, reconfiguration and deletion tests are run
+#### Set KUBECONFIG
+
+To run tests, you should have a running cluster, and your KUBECONFIG set.
 
-Example command to run the full test suite with 2 instances configured by the Windows Machine controller, and 1
-configured by the ConfigMap controller, skipping the deletion of all the nodes.
 ```shell script
-hack/run-ci-e2e-test.sh -s -m 2 -c 1
+export KUBECONFIG=<path to kubeconfig> 
 ```
-Please note that you do not need to run `hack/olm.sh run` before `hack/run-ci-e2e-test.sh`.
 
-#### Running e2e tests on platform-agnostic infrastructure
+The operator can now be deployed through one of these methods.
+- [Manually](#deploying-the-operator-manually)
+- [Using the OLM](#running-the-olm-hack-script)
+- [Using the e2e tests](#running-the-e2e-tests)
+- [Using the e2e tests on platform agnostic infrastructure](#running-e2e-tests-on-platform-agnostic-infrastructure)
+- [Using the official deployment method](#deploying-Using-the-official-deployment-method)
```

Comment:
Please remove this from here. You don't need to build anything for deploying using the official method.